cpp/inc/bond/core/bond.h

```diff
@@ -13,116 +13,116 @@ namespace bond
 
```

Comment:
Specifically which functions are you referring to? The order is usually determined based on the reasons I replied in above comment.